css/style.scss

```diff
@@ -922,9 +922,66 @@ input[type="password"] {
 				}
 			}
 		}
+		.talk-settings-button {
+			position: relative;
+			
+			.button {
+				cursor: pointer;
+				width: 50px;
+				height: 50px;
+				display: block;
+				background-color: transparent;
+				border: none;
+				margin: 0;
+				opacity: .7;
+				&:hover,
+				&:focus,
+				&:active {
+					opacity: 1;
+				}
+			}
+		}
 	}
 }
 
+.icon-speaker {
```

Comment:
instead do:
```scss
@include icon-black-white('menu-people', 'spreed', 1);
```

And you forgot to add the image.

Reply:
The image has been added.